src/test/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/database/jpa/mapper/induction/UpdatePreviousQualificationsEntityMapperTest.kt

```diff
@@ -16,12 +16,7 @@ import uk.gov.justice.digital.hmpps.educationandworkplanapi.app.database.jpa.ent
 
 class UpdatePreviousQualificationsEntityMapperTest {
 
-  private val mapper = PreviousQualificationsEntityMapperImpl().also {
-    PreviousQualificationsEntityMapper::class.java.getDeclaredField("qualificationEntityMapper").apply {
-      isAccessible = true
-      set(it, QualificationEntityMapperImpl())
-    }
-  }
+  private val mapper = PreviousQualificationsEntityMapper(QualificationEntityMapper())
```

Comment:
Easier to construct in tests now the mappers are standard classes with constructors to inject the dependency.
No need for this messy reflection any more